### PR TITLE
Simplify development server commads

### PR DESCRIPTION
**Pull request by kumar303**
Closes https://github.com/mozilla/addons-frontend/issues/2833

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,20 +34,20 @@ The easiest way to manage multiple node versions in development is to use
 ## Get started
 
 * type `yarn` to install all dependencies
-* type `yarn dev:amo:dev` to start a local dev server that connects to a
-  hosted development environment
+* type `yarn amo:stage` to start a local server that connects to a
```
Comment (kumar303) [inline README.md]: I think stage data is probably the best for newbies since it's cleaner and has icons (etc.)

File: README.md [hunk 2]
```diff
@@ -34,20 +34,20 @@ The easiest way to manage multiple node versions in development is to use
...
-| yarn dev:amo                | Start the dev server/proxy (for amo) using data from Docker |
-| yarn dev:amo:dev            | Start the dev server/proxy (for amo) using data from the dev server (https://addons-dev.allizom.org/) |
-| yarn dev:amo:no-proxy       | Start the dev server without a proxy (for amo) using data from Docker |
-| yarn dev:amo:stage          | Start the dev server/proxy (for amo) using data from the staging server (https://addons.allizom.org/) |
-| yarn dev:disco              | Start the dev server (for Discovery Pane) using data from Docker |
+| yarn amo                    | Start the dev server/proxy (for amo) using data from Docker |
+| yarn amo:dev                | Start the dev server/proxy (for amo) using data from the dev server (https://addons-dev.allizom.org/) |
+| yarn amo:no-proxy           | Start the dev server without a proxy (for amo) using data from Docker |
+| yarn amo:stage              | Start the dev server/proxy (for amo) using data from the staging server (https://addons.allizom.org/) |
+| yarn disco                  | Start the dev server (for Discovery Pane) using data from the dev server (https://addons-dev.allizom.org/) |
```
Comment (kumar303) [inline README.md]: In testing, I realized that disco still defaults to using dev data

File: README.md (hunk 1 above)
Comment (tofumatt) [inline README.md]: Agreed, actually. Data on -dev is actually kind of a mess.

File: README.md (hunk 2 above)
Comment (tofumatt) [inline README.md]: I feel like we should change that, but it can be a separate patch. I guess Disco data is pretty standard though.

Comment (kumar303) [inline README.md]: I think it's because we don't auto-generate featured Discopane add-ons in Docker so it would be a bit hard to set up

**Comment by kumar303** [comment]
@muffinresearch @tsl143 @aayushsanghavi heads up that when you next pull from master you'll need to run `yarn amo` instead of `yarn dev:amo` to start a local server. Also note that it's easier to develop with stage data now (which is the cleanest). You can simply run `yarn amo:stage`.
